test/test_jit.py

```diff
@@ -13142,6 +13142,30 @@ def checkModule(self, nn_module, args):
 
         return sm
 
+    @unittest.skipIf(PY2, "Class annotations are a thing in > 3.5")
+    def test_constants_with_final(self):
+        class M(torch.nn.Module):
+            # TODO: Use this (see below)
```

Comment:
what's the deal with these TODOs?

Reply:
This is Python 3 only syntax, including it in this test makes all the Python2  tests fail to parse the file